src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -659,9 +668,13 @@ private void testShutdownWhileRunning( boolean shouldThrow ) throws ClassNotFoun
             testDaemon.waitForCompletion( );
             assertTrue( testDaemon.hasRun( ) );
             Thread.sleep( 10L ); // leave some time to the daemon to exit
-                                 // executing daemons
+            // executing daemons
             assertEquals( 1, testDaemon.getStopCallNumber( ) );
         }
+        catch( InterruptedException | BrokenBarrierException | TimeoutException e )
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?